definitions/npm/@apollo/react-hooks_v3.x.x/flow_v0.104.x-/react-hooks_v3.x.x.js

```diff
@@ -0,0 +1,1048 @@
+// @flow
+declare module '@apollo/react-hooks' {
+  import type {ComponentType, Element, Node} from 'react';
+
+  /* start graphql types */
+  declare type DocumentNode = any;
+  declare type GraphQLError = any;
+  /* end graphql types */
+
+  /* start @apollo/react-common types */
+  declare export type ApolloProviderProps<TCache> = {
+    client: ApolloClient<TCache>,
+    children: Node | Node[] | null,
+  };
```

Comment:
Use either exact `{| |}` or inexact `{ ... }` objects.   
That's why the tests are failing.